website/addons/box/utils.py

```diff
@@ -78,6 +86,40 @@ def log(self, action, extra=None, save=False):
             self.node.save()
 
 
+def refresh_oauth_key(external_account, force=False):
```

Comment:
This doesn't look specific to Box - shouldn't it go somewhere in `website.oauth`?